projectgenerator/utils/qgis_utils.py

```diff
@@ -48,7 +52,7 @@ def get_first_index_for_layer_type(layer_type, group):
     return None
 
 
-def get_suggested_index_for_layer(layer, group):
+def get_suggested_index_for_layer(layer, group, ignore_node_names=[]):
```

Comment:
Better avoid mutable defaults (i.e. use `ignore_node_names=None`)
See https://docs.python-guide.org/writing/gotchas/#what-you-should-do-instead

Reply:
Thanks for the suggestion, should be fixed now.